ArmState rArm
ArmState lArm
float64 rArmVelocity
float64 lArmVelocity
bool useRight
bool useLeft